css-canvas-medical/styles.css

```diff
@@ -0,0 +1,242 @@
+body {
+  margin: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+.header-row {
+  display: flex;
+  width: 100%;
+  border-bottom: solid 2px #f2f4f5;
+  font-family: 'Avenir', sans-serif;
+}
+
+.nav-bar {
+  width: 50%;
+  display: flex;
+  color: #b9c2cb;
+  justify-content: space-evenly;
+  font-size: 12px;
+  margin-top: 10px;
+  font-weight: bolder;
+}
+
+.canvas-logo {
+  margin-top: 15px;
+  margin-left: 40px;
+  width: 50%;
+}
+
+.nav-bar,
+.canvas-logo {
+  margin-bottom: 15px;
+}
+
+.hero-image {
+  height: 75vh;
+  width: auto;
+  background-image: linear-gradient(transparent, rgba(0.7, 0.7, 0.7, 0.7)), url(images/hero.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin-top: 75px;
+  position: relative;
+  margin-left: 50px;
+  margin-right: 50px;
+}
+
+.hero-row {
+  width: 100%;
+  font-family: 'Tiempos Headline', sans-serif;
+}
+
+.hero-head-text {
+  color: white;
+  width: 40%;
+  align-items: center;
+  margin-left: 5rem;
+  position: absolute;
```

Comment:
If you are absolute positioning this element, why does it have margin? I may be mistaken, but it should be unnecessary.

Reply:
I used that to push it away from the edge a bit .. this is what it was like without margin 
![Screen Shot 2020-10-12 at 1 08 49 PM](https://user-images.githubusercontent.com/66848620/95786296-1ad42e80-0c8c-11eb-8b32-84455905fa55.png)

Reply:
Should i have just used left property to position it ?